exotic/exotic.py

```diff
@@ -2014,7 +2035,7 @@ def main():
                 for a, aper in enumerate(apers):
                     for an, annulus in enumerate(annuli):
                         if not np.isnan(psf_data['target'][i, 0]):
-                            aper_data["target"][i][a][an], aper_data["target_bg"][i][a][an] = aperPhot(imageData,
+                            aper_data["target"][i][a][an], aper_data["target_bg"][i][a][an] = aperPhot(imageData, 0,
```

Comment:
This is an API change for `aperPhot(...)` (note new parm). Are we sure that this was wrong or did the signature of the method also change? If the signature changed, can we be sure nothing else uses it?

Reply:
Not clear on why this is singled out as an API versus not - this is an application, not a library, so if we consider certain functions to be published/supported APIs versus not, I certainly do not see any convention by which this is able to be determined.   If we need to preserve this binding, wrapping it is an option

Reply:
Parts of it actually are imported (as a library) into internal applications, and potentially could be used that way outside, also. However, I don't think this part of the code is used that way by the Lab. Failsafe is that if anything breaks it can be pinned to the last version. Anyway, I was mostly considering any calls within the application to the same code. Also, the comment breaks the line so things look a little weird. Anyway, just a sanity check.